nx-X11/lib/src/Makefile.am

```diff
@@ -0,0 +1,368 @@
+		GetWMProto.c				\
+		globals.c				\
+		GrButton.c				\
+		GrKeybd.c				\
+		GrKey.c					\
+		GrPointer.c				\
+		GrServer.c				\
+		Host.c					\
+		Iconify.c				\
+		IfEvent.c				\
+		imConv.c				\
+		ImText16.c				\
+		ImText.c				\
+		ImUtil.c				\
+		InitExt.c				\
+		InsCmap.c				\
+		IntAtom.c				\
+		KeyBind.c				\
+		KeysymStr.c				\
+		KillCl.c				\
+		LiHosts.c				\
+		LiICmaps.c				\
+		LiProps.c				\
+		ListExt.c				\
+		LoadFont.c				\
+		LockDis.c				\
+		locking.c				\
+		LookupCol.c				\
+		LowerWin.c				\
+		Macros.c				\
+		MapRaised.c				\
+		MapSubs.c				\
+		MapWindow.c				\
+		MaskEvent.c				\
+		Misc.c					\
+		ModMap.c				\
+		MoveWin.c				\
+		NextEvent.c				\
+		OCWrap.c				\
+		OMWrap.c				\
+		OpenDis.c				\
+		ParseCmd.c				\
+		ParseCol.c				\
+		ParseGeom.c				\
+		PeekEvent.c				\
+		PeekIfEv.c				\
+		Pending.c				\
+		PixFormats.c				\
+		PmapBgnd.c				\
+		PmapBord.c				\
+		PolyReg.c				\
+		PolyTxt16.c				\
+		PolyTxt.c				\
+		PropAlloc.c				\
+		PutBEvent.c				\
+		PutImage.c				\
+		Quarks.c				\
+		QuBest.c				\
+		QuColor.c				\
+		QuColors.c				\
+		QuCurShp.c				\
+		QuExt.c					\
+		QuKeybd.c				\
+		QuPntr.c				\
+		QuStipShp.c				\
+		QuTextE16.c				\
+		QuTextExt.c				\
+		QuTileShp.c				\
+		QuTree.c				\
+		RaiseWin.c				\
+		RdBitF.c				\
+		RecolorC.c				\
+		ReconfWin.c				\
+		ReconfWM.c				\
+		Region.c				\
+		RegstFlt.c				\
+		RepWindow.c				\
+		RestackWs.c				\
+		RotProp.c				\
+		ScrResStr.c				\
+		SelInput.c				\
+		SendEvent.c				\
+		SetBack.c				\
+		SetClMask.c				\
+		SetClOrig.c				\
+		SetCRects.c				\
+		SetDashes.c				\
+		SetFont.c				\
+		SetFore.c				\
+		SetFPath.c				\
+		SetFunc.c				\
+		SetHints.c				\
+		SetIFocus.c				\
+		SetLocale.c				\
+		SetLStyle.c				\
+		SetNrmHint.c				\
+		SetPMask.c				\
+		SetPntMap.c				\
+		SetRGBCMap.c				\
+		SetSOwner.c				\
+		SetSSaver.c				\
+		SetState.c				\
+		SetStCmap.c				\
+		SetStip.c				\
+		SetTile.c				\
+		SetTSOrig.c				\
+		SetTxtProp.c				\
+		SetWMCMapW.c				\
+		SetWMProto.c				\
+		StBytes.c				\
+		StColor.c				\
+		StColors.c				\
+		StName.c				\
+		StNColor.c				\
+		StrKeysym.c				\
+		StrToText.c				\
+		Sync.c					\
+		Synchro.c				\
+		Text16.c				\
+		Text.c					\
+		TextExt16.c				\
+		TextExt.c				\
+		TextToStr.c				\
+		TrCoords.c				\
+		UndefCurs.c				\
+		UngrabBut.c				\
+		UngrabKbd.c				\
+		UngrabKey.c				\
+		UngrabPtr.c				\
+		UngrabSvr.c				\
+		UninsCmap.c				\
+		UnldFont.c				\
+		UnmapSubs.c				\
+		UnmapWin.c				\
+		VisUtil.c				\
+		WarpPtr.c				\
+		Window.c				\
+		WinEvent.c				\
+		Withdraw.c				\
+		WMGeom.c				\
+		WMProps.c				\
+		WrBitF.c				\
+		XlibAsync.c				\
+		XlibInt.c				\
+		Xrm.c					\
+		x11_trans.c				\
+		$(NULL)
+
+#
+# ========================= Extra stuff ============================
+#
+
+if THRSTUBS
+libNX_X11_la_SOURCES+=UIThrStubs.c
+endif
+
+x11datadir = @X11_DATADIR@
+x11data_DATA = XErrorDB
+
+EXTRA_DIST =						\
+	$(x11data_DATA)					\
```

Comment:
nonono, os2stubs-like file got removed everywhere. Even from Xorg Server. The libX11 still has support for that, because X.org provides more legacy support for the X11 libs than for the Xserver itself.